django/db/backends/oracle/introspection.py

```diff
@@ -17,39 +20,23 @@ class DatabaseIntrospection(BaseDatabaseIntrospection):
     # Maps type objects to Django Field types.
     @cached_property
     def data_types_reverse(self):
-        if self.connection.cx_oracle_version < (8,):
-            return {
-                cx_Oracle.BLOB: "BinaryField",
-                cx_Oracle.CLOB: "TextField",
-                cx_Oracle.DATETIME: "DateField",
-                cx_Oracle.FIXED_CHAR: "CharField",
-                cx_Oracle.FIXED_NCHAR: "CharField",
-                cx_Oracle.INTERVAL: "DurationField",
-                cx_Oracle.NATIVE_FLOAT: "FloatField",
-                cx_Oracle.NCHAR: "CharField",
-                cx_Oracle.NCLOB: "TextField",
-                cx_Oracle.NUMBER: "DecimalField",
-                cx_Oracle.STRING: "CharField",
-                cx_Oracle.TIMESTAMP: "DateTimeField",
-            }
-        else:
-            return {
-                cx_Oracle.DB_TYPE_DATE: "DateField",
-                cx_Oracle.DB_TYPE_BINARY_DOUBLE: "FloatField",
-                cx_Oracle.DB_TYPE_BLOB: "BinaryField",
-                cx_Oracle.DB_TYPE_CHAR: "CharField",
-                cx_Oracle.DB_TYPE_CLOB: "TextField",
-                cx_Oracle.DB_TYPE_INTERVAL_DS: "DurationField",
-                cx_Oracle.DB_TYPE_NCHAR: "CharField",
-                cx_Oracle.DB_TYPE_NCLOB: "TextField",
-                cx_Oracle.DB_TYPE_NVARCHAR: "CharField",
-                cx_Oracle.DB_TYPE_NUMBER: "DecimalField",
-                cx_Oracle.DB_TYPE_TIMESTAMP: "DateTimeField",
-                cx_Oracle.DB_TYPE_VARCHAR: "CharField",
-            }
+        return {
+            oracledb.DB_TYPE_DATE: "DateField",
+            oracledb.DB_TYPE_BINARY_DOUBLE: "FloatField",
+            oracledb.DB_TYPE_BLOB: "BinaryField",
+            oracledb.DB_TYPE_CHAR: "CharField",
+            oracledb.DB_TYPE_CLOB: "TextField",
+            oracledb.DB_TYPE_INTERVAL_DS: "DurationField",
+            oracledb.DB_TYPE_NCHAR: "CharField",
+            oracledb.DB_TYPE_NCLOB: "TextField",
+            oracledb.DB_TYPE_NVARCHAR: "CharField",
+            oracledb.DB_TYPE_NUMBER: "DecimalField",
+            oracledb.DB_TYPE_TIMESTAMP: "DateTimeField",
+            oracledb.DB_TYPE_VARCHAR: "CharField",
+        }
```

Comment:
You cannot remove the `cx_Oracle` < 8 branch, it's deprecated but still supported.

Reply:
Oops, forgot that. Just fixed.